model/components/commercial/appdynamics.yml

```diff
@@ -0,0 +1,126 @@
+    categories:
+      - commercial-services
+    capabilities:
+      aspects:
+        - tracing
+        - metrics
+        - logs
+    connections:
+      dataTo:
+        - grafana
+      dataFrom:
+        - appdynamics-agent
+        - appdynamics-sdks
+        - opentracing
+    isCommercial: true
+
+      # AppDynamics Collector
+  - id: appdynamics-collector
+    name: AppDynamics Controller
+    description: The AppDynamics Controller is the central element of the AppDynamics Monitoring Plattform. The AppDynamics Controller UI is the primary interface you will use to monitor, troubleshoot, and analyze your entire application landscape from backend infrastructure/servers to the end user's application. 
+    link: https://docs.appdynamics.com/display/LATEST/AppDynamics+Application+Performance+Monitoring+Platform
+    twitter: AppDynamics
+    logo: appdynamics
+    categories:
+      - collector
+      - data-processing
+      - storage
+      - dashboarding
+      - alerting
+      - visualisation
+    capabilities:
+      os:
+        - linux
+        - Windows
+        - SaaS
+      tech:
+        - AWS
+        - Azure
+        - Google Cloudplattform
+      aspects:
+        - tracing
+        - metrics
+        - analytics    
+    connections:
+      dataTo:
+        - grafana
+      dataFrom:
+        - appdynamics-sdks
+        - appdynamics-agent
+        - opentracing    
+    isCommercial: true
+
+  # AppDynamics APM Agent
+  - id: appdynamics-agent
+    name: AppDynamics Agent
+    description: AppDynamics Application Performance Monitoring (APM), a component of the AppDynamics platform, gives you end-to-end visibility into the performance of your applications. The main goal of our agents is to have the lowest overhead possible on the system and during installation. Monst of the AppDynamics agents can be integrated without any code changes.
+    link: https://docs.appdynamics.com/display/LATEST/Application+Monitoring
+    twitter: AppDynamics
+    logo: appdynamics
+    categories:
+      - agent
+    capabilities:
+      os:
+        - linux
+        - windows
+        - MacOS
+      aspects:
+        - tracing
+        - metrics
+        - analytics
+        - eum
+      tech:
+        - java
+        - .net
+        - php
+        - python
+        - nodejs
+        - SAP ABAP
+        - IBM Integration Bus
+        - Webserver
+        - Browser Monitoring Java Script 
+    connections:
+      dataFrom:
+        - opentracing
+        - appdynamics-agent
+    isCommercial: true
+
+  # AppDynamics SDKs
+  - id: appdynamics-sdks
+    name: AppDynamics SDKs
+    description: AppDynamics Application Performance Monitoring (APM), a component of the AppDynamics platform, gives you end-to-end visibility into the performance of your applications. The main goal of our agents is to have the lowest overhead possible on the system and during installation. Monst of the AppDynamics agents can be integrated without any code changes.
```

Comment:
same description as for the Agent. Please provide a specific description for the SDK.